### DISCUSSION TITLE
Convert transformed RGB image back to RGB

### DISCUSSION THREAD

**Discussion by ary1111** [Project-MONAI/MONAI#7830] [Q&A]
I'm working through the 2D latent diffusion model tutorial for a colored set of images and I'm not sure what I need to do to my images to get them back to being RGB or if I need to do a different transform before training. This is the transform I apply to the training set:

```
train_transforms = transforms.Compose(
    [
        transforms.LoadImaged(keys=["image"]),
        transforms.EnsureChannelFirstd(keys=["image"]),
        transforms.ScaleIntensityRanged(keys=["image"], a_min=0.0, a_max=255.0, b_min=0.0, b_max=1.0, clip=True),
        transforms.CropForegroundd(keys=["image"], source_key="image", select_fn=lambda x: x > 0.1),
        transforms.Resized(keys=["image"], spatial_size=[image_size, image_size]),
    ]
)
```
This is how the generated images look and I would like them to be RGB:
![image](https://github.com/Project-MONAI/MONAI/assets/18470260/e5ec5c15-b47d-40ac-94c4-80fe9434e224)

**Comment by KumoLiu** [comment]
Hi @ary1111, since you applied intensity scaling to normalize the pixel values to the range [0, 1] during training, you need to inverse this operation to bring the pixel values back to the original range of [0, 255] for RGB images.
